packages/manager/apps/cloud/client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.html

```diff
@@ -41,14 +41,17 @@
 >
     <div class="row">
         <div class="col-xs-3">
-            <div class="btn-group cloud-compute-infra-primary-actions">
+            <div
+                data-uib-dropdown
```

Comment:
Why not use the ui-kit dropdown ?

Reply:
Because it's only a fix for now (check the goal on the description ;) )